_data-prepper/pipelines/configuration/processors/add-entries.md

```diff
@@ -18,6 +18,10 @@ You can configure the `add_entries` processor with the following options.
 | :--- | :--- | :--- |
 | `entries` | Yes | A list of entries to add to an event. |
 | `key` | Yes | The key of the new entry to be added. Some examples of keys include `my_key`, `myKey`, and `object/sub_Key`. |
+| `metadata_key` | Yes | The key for the new metadata attribute. Argument must be a literal string key and not a `JsonPointer`. Either one string key or `metadata_key` is required. |
```

Comment:
Let's remove the backticks from around "JsonPointer."

Reply:
Changed as suggested